src/libraries/System.Private.CoreLib/src/System/IO/Path.Unix.iOS.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace System.IO
+{
+    public static partial class Path
+    {
+        private static string s_defaultTempPath = string.Empty;
```

Comment:
```suggestion
        private static string? s_defaultTempPath = null;
```

Reply:
Yeah without this it won't assign the return value from SearchPathTempDirectory() to the variable